src/components/Error.astro

```diff
@@ -0,0 +1,60 @@
+<script src="https://harshkhandeparkar.github.io/typoist/dist/typoist.min.js"></script>
+<div class="center">
+  <h1 class="type-box" id="error-desc"></h1>
+</div>
+
+<script>
+  window.onload = () => {
+    const typeIt = (selector, text, options) => {
+      const textArea = document.querySelector(selector);
+
+      const typoist = new Typoist.Typoist({
+        ...options,
+        appendFunction: (character) =>
+          new Promise((resolve) => {
+            textArea.innerText += character;
+            resolve();
+          }),
+        deleteFunction: () =>
+          new Promise((resolve) => {
+            textArea.innerText = textArea.innerText.slice(0, -1);
+            resolve();
+          }),
+      });
+
+      typoist.type(text);
+    };
+
+    typeIt("#error-desc", "404 Error, Webpage not Found", { speed: 8, mistakeProbability: 0 });
+  };
+</script>
+
+<style>
+  .center {
```

Comment:
Can you use a monospace font?

Reply:
i wanted it to look like a bash output
I have some ideas for it ill add it later

Reply:
:nam: